app/src/test/java/ch/epfl/sdp/items/ItemsTest.java

```diff
@@ -88,11 +88,11 @@ public void scanTest() {
 
     @Test
     public void coinTest() {
-        PlayerManager.getCurrentUser().money = 0;
+        PlayerManager.getCurrentUser().removeMoney(PlayerManager.getCurrentUser().getMoney());
```

Comment:
This would've been much easier with a setter, but for now this works

Reply:
With a setter, we lose all safety on the money of the Player